packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -1342,6 +1367,20 @@ Adds an OAuth 2.0 client application. This application is only available to the
   * The following properties can also be configured in the App Wizard and on the **General** tab in the Admin Console: `tos_uri`, `policy_uri`, and `logo_uri` and can be set using the [Dynamic Client Registration API](/docs/reference/api/oauth-clients/).
   * The `consent_method` property can be configured in the App Wizard and on the **General** tab in the Admin Console, but cannot be set using the Dynamic Client Registration API.
 
+### Idp Initiated Login object
+
+The Idp Initiated Login object is used to configure what, if any, Idp Initiated Login flows an OAuth Client supports.
+
+| Property      | Description                                           | DataType                   | Nullable |
+| ------------- | ----------------------------------------------------- | -------------------------- | -------- | 
+| mode          | What mode to use for Idp Initiated Login              | `DISABLED`, `SPEC`, `OKTA` | FALSE    |
+| default_scope | What scopes to use for the request when mode = `OKTA` | List of String             | TRUE     |
+
+* When `mode` is `DISABLED` the client does not support Idp-Initiated Login
+* When `mode` is `SPEC` the client will be redirected to the Relying Party's initiate_login_uri as defined in the [OpenId Connect spec](https://openid.net/specs/openid-connect-core-1_0.html#ThirdPartyInitiatedLogin)
+* When `mode` is `OKTA` the tokens will be directly sent to the Relying Party. This corresponds the "Okta Simplified" option in the Administrator Console.
```

Comment:
```suggestion
* When `mode` is `OKTA`, the tokens are directly sent to the relying party. This corresponds with the **Okta Simplified** option in the Administrator Console.
```

Reply:
The spec that we are linking to here capitalizes `Relying Party`, and it's equivalent to `Service Provider` which we are capitalizing elsewhere in this document & PR. Should I still make it lower-case?